odl/tomo/util/source_detector_shifts.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2014-2020 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Source and detector shifts for divergent beam geometries."""
+
+from __future__ import print_function, division, absolute_import
+import numpy as np
+from odl.discr import uniform_discr_frompartition
+from odl.discr.discr_utils import nearest_interpolator
+
+__all__ = ('flying_focal_spot')
+
+
+def flying_focal_spot(angle, apart, shifts):
```

Comment:
Could you add a test for this in particular?

Reply:
What do you mean? Tests in geometry_tests use this function, since it is the only implemented option

Reply:
I agree, but they test the whole thing. It might be good to have a test for this in isolation to help track down errors.

Reply:
The other option, if this is not intended to be used externally, is to make this a private function in the cone beam geometry file.